orchestrator/utils/enrich_process.py

```diff
@@ -11,14 +11,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Optional
+from typing import Dict, Optional
 
-from orchestrator.db import ProcessTable, SubscriptionTable
-from orchestrator.workflow import ProcessStat, Step
+from more_itertools import first
+
+from orchestrator.db import ProcessStepTable, ProcessTable, SubscriptionTable
+from orchestrator.utils.get_updated_properties import get_dict_updates
+from orchestrator.workflow import ProcessStat, Step, StepStatus
 from pydantic_forms.core import generate_form
 
 
-def format_subscription(subscription: SubscriptionTable) -> dict:
+def format_subscription(subscription: SubscriptionTable) -> Dict:
```

Comment:
We require python >= 3.9. `dict` is equally powerful as `Dict` and preferred

Reply:
ye, thought 3.9 didn't like it....